be/src/exec/parquet_reader.cpp

```diff
@@ -223,7 +223,7 @@ Status ParquetReaderWrap::column_indices(const std::vector<SlotDescriptor*>& tup
             for (auto index : iter->second) {
                 _parquet_column_ids.emplace_back(index);
             }
-        } else {
+        } else if (!_enable_flexible_column_mapping) {
```

Comment:
How about other file formats besides Parquet?

Reply:
This PR only handles parquet. I'll submit several PRs to handle other formats.